docs/intentReaderHandlers/RunTestHandler.md

```diff
@@ -0,0 +1,304 @@
+# RunTestHandler 
+
+## Overview
+
+RunTestHandler is invoked when the task specified in the intent has the value "runTest". This handler is used to run sanity tests in FCA. Whent this handler is invoked, it will loop through all the apis in openRpc schema, invoke the apis and perform schema validations on the responses. Once the execution is over, a test report is generated. For more about execution [here](../Execution.md#sanity-test-execution) and for reporting [here](../Reporting.md#supported-ways-of-retrieving-reports) 
+
+It performs the following actions :
+1. Parse the input message received to fetch the action and parameters.
+2. Loop through the apis in openRpc. 
+3. Save the API response/error and perform schema validation.
+4. Format the result and send the response back to IntentReader.
```

Comment:
Formats the result and sends the response back to IntentReader.

Reply:
updated for all handlers